taxonomy/utils.py

```diff
@@ -253,12 +253,13 @@ def is_skill_blacklisted(key_or_uuid, skill_id, product_type):
     return skill_model.objects.filter(**kwargs).exists()
 
 
-def get_whitelisted_product_skills(key_or_uuid, product_type, prefetch_skills=True):
+def get_whitelisted_product_skills(key_or_uuid, product_type='Course', prefetch_skills=True):
```

Comment:
Should create a follow up task to not use hardcoded product_type string values, rather we will create a new Enum of ProductType and use that everywhere

Reply:
It has been addressed.